lib/opencog.conf

```diff
@@ -73,22 +73,22 @@ ANSI_ENABLED	       = true
 # Cogserver in OSX will automatically change .so extension to .dylib
 # if .so exists.
 MODULES               = opencog/cogserver/server/libbuiltinreqs.so,
-                        opencog/modules/libPersistModule.so,
-                        opencog/modules/libPersistZmqModule.so,
-                        opencog/modules/libQueryModule.so,
+                        opencog/cogserver/modules/libPersistModule.so,
+                        opencog/cogserver/modules/libPersistZmqModule.so,
+                        opencog/cogserver/modules/libQueryModule.so,
                         opencog/cogserver/shell/libscheme-shell.so,
```

Comment:
These changes are not correct, because they only work if you start the cogserver in the build tree. If you don't have a build tree available, then this config file will fail to load these modules.

This is the same problem that Amen is trying to solve by adding symlinks.